### ISSUE TITLE
Not quite an issue with this but soldering reset pin

### ISSUE THREAD

**Issue by fusionstream** [e-tinkers/ESP-01_with_ESP-Now#1]
Hi there. Wondering if you've considered performing the soldering hack for the esp01 to enable waking up from deepSleep.

If you have, do you have any tips? Or if you sent it to a shop to get it done, let me know where. I believe we're from the same area :)

I have maybe 5 or 10 esp01s which are almost completely useless to me because I can't effect a viable deepSleep solution. Without a deepsleep scenario, I find the d1 minis to be far more effective.

Also, thank you very much for this sketch. It's concise and elegant.

**Comment by e-tinkers** [comment]
I did once or twice in the past (see picture). It is depend on the soldering skill and the need of a magnifying equipment in order to get it done. It is not very easy but also not very difficult. I'm an electronics guy and holding a soldering iron since 12 years and for more than 40 years...

![IMG_2025](https://github.com/e-tinkers/ESP-01_with_ESP-Now/assets/32047718/f3a32533-f990-4628-8f03-df6b074a2283)

I'm like you have several EPS-01s, but I only modified two because in most of the cases whenever I need deep sleep, I would need battery charging circuit and others, this often would justify to make a custom PCB like this [project](https://github.com/e-tinkers/esp8266_ntp_clock_pcb) so it is better to use an ESP-12S.
